packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -851,6 +884,9 @@ export class JobService {
     }
 
     jobEntity.chainId = chainId;
+    jobEntity.reputationOracle = reputationOracle;
+    jobEntity.exchangeOracle = exchangeOracle || null;
```

Comment:
To avoid errors, for example, when the list of oracles returned by the subgraph does not contain one that matches the type of task requested by the user.

Reply:
But if we set them to null, when we try to call setup function of the escrow, it will fail

Reply:
Yes, you are right, I agree with that. Then I will remove the zero value.